### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by beanpole135**
Add all the usage/documentation on sysup to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,122 @@
...
+Only **one** of these arguments may be used at a time.
+- **-check**
+   - Check for updates
+- **-update**
+   - Start performing updates
+- **-list-trains**
+   - List all the pkg "trains" that the system is aware of (defined in "/usr/local/etc/sysup.json")
+- **-change-train TRAIN_NAME**
+   - Reconfigure the package repository files to point to the designated TRAIN_NAME.
+   - ***WARNING*** This will remove *all* package repository configuration files on the system and create a single "/etc/pkg/train.conf" file containing the configuration for the desired package train.
```
Comment (kmoore134) [inline README.md]: Should be /etc/pkg/Train.conf

File: README.md [hunk 2]
```diff
@@ -1,2 +1,122 @@
...
+- **-bename NAME**
+   - Use "NAME" for the new boot environment that will be created.
+   - A boot environment with "NAME" must *not* already exist, otherwise sysup will return an error.
+   - Default Value: sysup with automatically generate a unique boot environment name using a date/time stamp.
+      - Example of auto-generated BE name: "2018-11-27-14-34-26"
+- **-fullupdate**
+   - Force a "full" update of all packages (including kernel/world).
+   - Default Value: This is automatically determined based on whether the base packages (kernel/world) are tagged as newer on the package repository.
+- **-stage2**
+   - Start stage2 of an update (installing non-base packages)
```
Comment (kmoore134) [inline README.md]: Should be (Installing non-kernel package updates)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,122 @@
...
+**WARNING** : The manifest contains information on both available *and* obsolete trains. If a package repo is removed, then the train should be marked as depricated *but left in the manifest*. Removing a train from the manifest may result in unexpected behavior on client systems that are set to follow that train.
+
+### Example Manifest File
+```
+{
+  "trains" : [
+    {
+      "name" : "TRAIN_NAME",
+      "description" : "TRAIN_DESCRIPTION",
+      "depricated" : false,
```
Comment (kmoore134) [inline README.md]: Spelling error: 'deprecated'

**Review by kmoore134 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (beanpole135) [reply README.md]: Fixed!

File: README.md (hunk 2 above)
Comment (beanpole135) [reply README.md]: Fixed!

File: README.md (hunk 3 above)
Comment (beanpole135) [reply README.md]: Fixed!

**Comment by beanpole135** [comment]
Fixed all requested changes

**Review by kmoore134 — APPROVED** [review]

**Pull request merged**